mlir/test/Dialect/LLVMIR/mem2reg-dbginfo.mlir

```diff
@@ -36,17 +36,17 @@ llvm.func @block_argument_value(%arg0: i64, %arg1: i1) -> i64 {
   // CHECK-NOT: = llvm.alloca
   %1 = llvm.alloca %0 x i64 {alignment = 8 : i64} : (i32) -> !llvm.ptr
   // CHECK-NOT: llvm.intr.dbg.declare
-  llvm.intr.dbg.declare #di_local_variable = %1 : !llvm.ptr
+  llvm.intr.dbg.declare #di_local_variable #llvm.di_expr<[]> = %1 : !llvm.ptr
```

Comment:
suggestion: can you omit the exprs if they're empty?

Reply:
oh sure I can try that